darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -69,6 +92,52 @@ def __init__(
             f"{regression_model.lags}",
         )
 
+        raise_if(
+            regression_train_num_samples > 1 and not self._models_are_probabilistic(),
+            "`regression_train_num_samples` is greater than 1 but the `RegressionEnsembleModel` "
+            "contains at least one non-probabilistic forecasting model.",
+            logger,
+        )
+
+        # check the reduction method
+        supported_reduction = ["mean", "median"]
+        if isinstance(regression_train_samples_reduction, float):
+            # this is already checked by `ts.quantile()`, maybe too redundant
+            raise_if(
+                regression_train_samples_reduction > 1.0
+                or regression_train_samples_reduction < 0,
+                f"`regression_train_samples_reduction` should be comprised between "
+                f"0 and 1 ({regression_train_samples_reduction}).",
+                logger,
+            )
+        elif isinstance(regression_train_samples_reduction, str):
+            raise_if(
+                regression_train_samples_reduction not in supported_reduction,
+                f"`regression_train_samples_reduction` should be one of {supported_reduction}, "
+                f"received ({regression_train_samples_reduction})",
+                logger,
+            )
+        else:
+            logger.exception(
+                f"`regression_train_samples_reduction` type not supported "
+                f"({regression_train_samples_reduction}). Must be `float` "
+                f" or one of {supported_reduction}."
+            )
+
+        if (
+            regression_train_num_samples
+            * regression_train_n_points
+            * len(forecasting_models)
+            > SAMPLES_WARNING_THRESHOLD
```

Comment:
This was only required for the vertical stacking of samples and can be removed now